packages/grafana-ui/src/utils/handleReducedMotion.ts

```diff
@@ -0,0 +1,16 @@
+import { CSSInterpolation } from '@emotion/css';
+
+/**
+ * @param styles - Regular styles.
+ * @param reducedMotionStyles - Styles to apply when `prefers-reduced-motion` is enabled.
+ * If @param reducedMotionStyles is not provided, there won't be any animation or transition shown.
```

Comment:
```suggestion
 * Applies one of `styles` or `reducedMotionStyles` depending on a users `prefers-reduced-motion` setting.
```

Reply:
But we are not explaining what happens if `reducedMotionStyles` is not set, right? If there is no value then the animation, when `prefers-reduced-motion` is enabled, won't be shown.

Reply:
yeah! Like it 😉